resources/langs/nheko_de.ts

```diff
@@ -6,89 +6,89 @@
     <message>
         <location filename="../../src/dialogs/CreateRoom.cc" line="32"/>
         <source>CANCEL</source>
-        <translation type="unfinished"></translation>
+        <translation>ABBRECHEN</translation>
     </message>
     <message>
         <location filename="../../src/dialogs/CreateRoom.cc" line="43"/>
         <source>Name</source>
-        <translation type="unfinished"></translation>
+        <translation>Titel</translation>
     </message>
     <message>
         <location filename="../../src/dialogs/CreateRoom.cc" line="46"/>
         <source>Topic</source>
-        <translation type="unfinished"></translation>
+        <translation type="unfinished">Thema/Beschreibung</translation>
     </message>
     <message>
         <location filename="../../src/dialogs/CreateRoom.cc" line="49"/>
         <source>Alias</source>
-        <translation type="unfinished"></translation>
+        <translation>Alias</translation>
     </message>
     <message>
         <location filename="../../src/dialogs/CreateRoom.cc" line="57"/>
         <source>Room Visibility</source>
-        <translation type="unfinished"></translation>
+        <translation>Raum Sichtbarkeit</translation>
```

Comment:
I would prefer a single word - or to connect those two with "-"

Reply:
Going for a single word